lib/linguist/version.rb

```diff
@@ -1,3 +1,3 @@
 module Linguist
-  VERSION = "6.4.0"
+  VERSION = File.read(File.expand_path("../VERSION", __FILE__)).strip
```

Comment:
Nice. One less hack for the release script I've been slowly working on.

Reply:
Sweet!